DFC.App.DiscoverSkillsCareers/ViewModels/Assessment/AssessmentReferenceGetResponse.cs

```diff
@@ -13,7 +13,7 @@ public class AssessmentReferenceGetResponse
         [Required(ErrorMessage = "Enter a phone number")]
         [RegularExpression(@"^(((\+44\s?\d{4}|\(?0\d{4}\)?)\s?\d{3}\s?\d{3})|((\+44\s?\d{3}|\(?0\d{3}\)?)\s?\d{3}\s?\d{4})|((\+44\s?\d{2}|\(?0\d{2}\)?)\s?\d{4}\s?\d{4}))(\s?\#(\d{4}|\d{3}))?$", 
             ErrorMessage = "Enter a mobile phone number, like 07700 900 982.")]
-        [DataType(DataType.PhoneNumber)]
+        [DataType(DataType.PhoneNumber, ErrorMessage = "Enter a phone number")]
```

Comment:
Did you get the regex from here? -> https://skillsfundingagency.atlassian.net/wiki/spaces/DFC/pages/1919091155/CUI+field+validation+rules